vividus/src/main/java/org/vividus/configuration/VaultStoredPropertiesProcessor.java

```diff
@@ -66,6 +69,16 @@ protected String processValue(String propertyName, String partOfPropertyValueToP
                 ));
     }
 
+    @Override
+    public String processProperty(String propertyName, String propertyValue)
+    {
+        if (propertyValue.matches("(?<!AZURE_KEY_|HASHI_CORP_)VAULT\\((.+?)\\)"))
```

Comment:
re-use `propertyPattern` here

Reply:
I can't reuse `propertyPattern` here, because it works for both `VAULT(...)` and `HASHI_CORP_ VAULT(...)`.
But log should be printed only for `VAULT(...)`